it/migrations/MovementsChangeLogSpec.scala

```diff
@@ -28,103 +28,103 @@ import org.scalatest.BeforeAndAfterAll
-
-      _ <- db.drop()
-
-      coll = db.collection[JSONCollection](ArrivalMovementRepository.collectionName)
-
-      _ <- coll.insert.many {
-        for (id <- arrivalIds)
-          yield Json.obj(
-            "_id"                      -> id.index,
-            "channel"                  -> ChannelType.web.toString,
-            "eoriNumber"               -> eori,
-            "movementReferenceNumber"  -> Random.alphanumeric.take(20).mkString,
-            "created"                  -> LocalDateTime.of(2021, 7, 15, 12, 12),
-            "updated"                  -> LocalDateTime.of(2021, 7, 15, 12, 13),
-            "lastUpdated"              -> LocalDateTime.of(2021, 7, 15, 12, 13),
-            "nextMessageCorrelationId" -> 2,
-            "messages" -> Json.arr(
-              Json.obj(
-                "dateTime"             -> LocalDateTime.of(2021, 7, 15, 12, 12),
-                "received"             -> LocalDateTime.of(2021, 7, 15, 12, 12),
-                "messageType"          -> MessageType.ArrivalNotification.toString,
-                "message"              -> NodeSeq.fromSeq(Seq(<CC007A></CC007A>)),
-                "status"               -> MessageStatus.SubmissionSucceeded.toString,
-                "messageCorrelationId" -> 1
-              ),
-              Json.obj(
-                "dateTime"             -> LocalDateTime.of(2021, 7, 15, 12, 12),
-                "received"             -> LocalDateTime.of(2021, 7, 15, 12, 12),
-                "messageType"          -> MessageType.UnloadingPermission.toString,
-                "message"              -> NodeSeq.fromSeq(Seq(<CC043A></CC043A>)),
-                "messageCorrelationId" -> 1
-              ),
-              Json.obj(
-                "dateTime"             -> LocalDateTime.of(2021, 7, 15, 12, 13),
-                "received"             -> LocalDateTime.of(2021, 7, 15, 12, 12),
-                "messageType"          -> MessageType.UnloadingRemarks.toString,
-                "message"              -> NodeSeq.fromSeq(Seq(<CC044A></CC044A>)),
-                "messageCorrelationId" -> 1
-              )
-            )
-          )
-      }
-    } yield ()
-
-    insertArrivals.futureValue
-  }
-
-  "MovementsChangeLog" - {
-    "addMessageIdToMessages" - {
-      "should add message ID to messages in the arrivals collection" in {
-        val repo   = app.injector.instanceOf[ArrivalMovementRepository]
-        val runner = app.injector.instanceOf[MigrationRunnerImpl]
-
-        runner.runMigrations().futureValue
-
-        arrivalIds.foreach {
-          arrivalId =>
-            val arrival = repo.get(arrivalId).futureValue.value
-            arrival.messages.mapWithIndex {
-              case (message, index) =>
-                message.messageId.value mustBe (index + 1)
-            }
-        }
-      }
-    }
-  }
-}
+//class MovementsChangeLogSpec { // extends ItSpecBase with IntegrationPatience with BeforeAndAfterAll with GuiceOneAppPerSuite {
```

Comment:
Accidental commented out code?

Reply:
Not accidental as such but is not required and will be removed.